test/model/recent_senders_checks.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:checks/checks.dart';
+import 'package:zulip/model/recent_senders.dart';
+
+extension RecentSendersChecks on Subject<RecentSenders> {
+  Subject<Map<int, Map<int, MessageIdTracker>>> get streamSenders =>
+    has((rs) => rs.streamSenders, 'streamSenders');
+
+  Subject<Map<int, Map<String, Map<int, MessageIdTracker>>>> get topicSenders =>
+    has((rs) => rs.topicSenders, 'topicSenders');
```

Comment:
nit: follow maximally-boring, uniform pattern found on other checks-extensions:
```suggestion
  Subject<Map<int, Map<int, MessageIdTracker>>> get streamSenders => has((rs) => rs.streamSenders, 'streamSenders');
  Subject<Map<int, Map<String, Map<int, MessageIdTracker>>>> get topicSenders => has((rs) => rs.topicSenders, 'topicSenders');
```